query-engine/connector-test-kit-rs/query-engine-tests/tests/new/interactive_tx.rs

```diff
@@ -450,7 +451,7 @@ mod itx_isolation {
 
     #[connector_test(exclude(MongoDb))]
     async fn casing_doesnt_matter(mut runner: Runner) -> TestResult<()> {
-        let tx_id = runner.start_tx(5000, 5000, Some("sErIaLiZaBlE".to_owned())).await?;
+        let tx_id = runner.start_tx(5000, 5000, Some("rEpEaTaBlErEaD".to_owned())).await?;
```

Comment:
Why this change?

Reply:
Considering that tidb does not support the Serializable isolation level, perhaps I should add two additional test cases of TiDB instead of modifying the original test case.

Reply:
Yes, that would make more sense. Probably worth testing repeatable reads anyway.

(Curious why this does not really differential between serializable and repeatable it seems)